### ISSUE TITLE
Convert game to Pixi.JS

### ISSUE THREAD

**Issue by dimtion** [OtterLine/innovGame#6]
I think with more than 10 players we will run in performances issues especially on mobile devices.

Either we find a good optimization (like not updating every object at each run)
Either we change technology to use something like canvas or [pixi.js](http://www.pixijs.com/)

What do you think ?

**Comment by nimag42** [comment]
Yes of course, the first version of the game is clearly not well sized for >2 players, it was just a first attempt.
But in v0.3 we should consider one of these two techno.

Personnaly, I'm rather onto pixi.js that I know a bit more

**Comment by nimag42** [comment]
While converting game to a client side part, we use Pixi.JS for more performance
